libexec/cli/list.help

```diff
@@ -0,0 +1,32 @@
+USAGE: singularity [...] list [list options...]
+
+List all running named instances
```

Comment:
+1 for:

- status `singularity status container.img mysql` --> show me container instance status
- ps`singularity ps`--> show me all running processes. `ps` coincides with use of docker-compose and docker, so my preference is for ps to be associated with listing processes.
- `list` I think we should save for something in the future that will be highly used, like listing containers in some registry, or that match some filter, etc.